pkg/windows/types.go

```diff
@@ -0,0 +1,59 @@
+// +build windows
+
+package windows
+
+import "github.com/rancher/k3s/pkg/daemons/config"
+
+type CniConfig struct {
+	NodeConfig   *config.Node
+	CalicoConfig *CalicoConfig
+	NetworkName  string
+	BindAddress  string
+}
+
+type FelixConfig struct {
```

Comment:
Why are these defined here - can we import them from an external package?

Reply:
These types don't really exist upstream in a format that makes sense to use.